src/tools/meson-flex-wrapper

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument('--flex', dest='flex_command', help='flex executable to run', type=str)
+parser.add_argument('--perl', dest='perl_command', help='perl executable to run', type=str)
+parser.add_argument('--fix_warning_script', dest='fix_warning_script', help='script for fixing warnings', default='')
+parser.add_argument('-o', dest='output_file', help='output file')
+
+args, rest = parser.parse_known_args()
```

Comment:
Might be nicer to delineate arguments going directly to flex with -- or such? I.e. flex-wrapper --flex ... --perl ... -o ... -- -b -b -CFe

Reply:
I was trying to address that but it seems we need to split args from `--`, I couldn't find a way to implement that without splitting args. And it seems, recommended way to do that is `parse_known_args()` from https://docs.python.org/3.4/library/argparse.html#partial-parsing.